indrasetiawan.is-not-a.dev.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "check this out",
+
+  "domain": "is-not-a.dev",
+  "subdomain": "indrasetiawan",
+
+  "owner": {
+    "repo": "https://github.com/Gand0r/Gand0r.github.io",
+    "email": "portgash.the.ace@gmail.com"
+  },
+
+  "record": {
+    "CNAME": "indrasetiawan.is-not-a.dev"
```

Comment:
CNAME should have the name of the record and the value "being where it's going to be pointed to".

Reply:
Can i point to my github page like https://gand0r.github.io/ ?